client/state/comments/reducer.js

```diff
@@ -108,6 +113,7 @@ export function items( state = {}, action ) {
 
 	return state;
 }
+items.schema = commentsSchema;
```

Comment:
mostly that we never update the comment list. so a refresh wouldn't get you the newest comments if we persisted them... if we add in updating / fetching new comments, persisting would make a ton of sense.